src/ThemeLayer/appTheme.js

```diff
@@ -0,0 +1,30 @@
+import { createMuiTheme } from "@material-ui/core/styles";
+
+/**
+ * palette
+ *
+ * @type {Object} overrides material-ui palette
+ */
+const palette = {
```

Comment:
I would like these to be editable in the future. The user should be able to select the background color.

Reply:
If we follow material-ui theming and color rules we easily can make it editable. You can see it in action here in [material-ui docs](https://material-ui.com/customization/color/#color-tool).